src/main/java/com/toptrumps/core/card/Card.java

```diff
@@ -37,14 +39,23 @@ public Attribute getAttributeByName(String name) {
                 .findFirst()
                 .orElse(null);
     }
+    private void findHighestAttributes() {
+        Supplier<Attribute> highestAttribute = () -> Collections.max(attributes);
```

Comment:
This is the same logic from a couple of other places. Should we consider creating a new ```utils``` class?